### DISCUSSION TITLE
Can you have multiple leaders, or static trackers?

### DISCUSSION THREAD

**Discussion by tomjones1977** [nzymedefense/nzyme#501] [Questions] [answered]
If you have a large enough Wi-Fi estate a single leader wouldn't cover enough physical area.  It seems logical that you could use a tracker to extend the range, but can you use multiple trackers to monitor, or set up multiple leaders feeding one database?

**Comment by lennartkoopmann** [comment, marked as answer]
Hi! Not yet but it is definitely on the roadmap. One of the next big things coming.
